docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,765 @@
+  ...........................................................................................................................................+++
+  e is 65537 (0x10001)
+  Using configuration from /opt/dirac/etc/grid-security/ca/openssl_config_ca.cnf
+  Check that the request matches the signature
+  Signature ok
+  Certificate Details:
+          Serial Number: 4097 (0x1001)
+          Validity
+              Not Before: Apr 10 14:48:31 2019 GMT
+              Not After : Apr  9 14:48:31 2020 GMT
+          Subject:
+              countryName               = ch
+              organizationName          = DIRAC
+              organizationalUnitName    = DIRAC CI
+              commonName                = ciuser
+              emailAddress              = lhcb-dirac-ci@cern.ch
+          X509v3 extensions:
+              X509v3 Basic Constraints:
+                  CA:FALSE
+              X509v3 Subject Key Identifier:
+                  98:BB:F0:A8:96:4F:80:C8:3E:21:60:5E:FD:17:4E:34:97:EF:31:17
+              X509v3 Authority Key Identifier:
+                  keyid:33:F0:C8:60:6D:6B:52:BD:E9:A7:FA:57:27:72:5A:5D:7E:43:12:ED
+
+              X509v3 Key Usage: critical
+                  Digital Signature, Non Repudiation, Key Encipherment
+              X509v3 Extended Key Usage:
+                  TLS Web Client Authentication
+              Netscape Comment:
+                  OpenSSL Generated Client Certificate
+  Certificate is to be certified until Apr  9 14:48:31 2020 GMT (365 days)
+
+  Write out database with 1 new entries
+  Data Base Updated
+
+To finish, time to exit the subshell::
+
+  exit
+
+
+At this point, you should find:
+
+ * The CA in `/opt/dirac/etc/grid-security/certificates`::
+
+    [dirac@dirac-tuto caUtilities]$ ls /opt/dirac/etc/grid-security/certificates/
+    855f710d.0  ca.cert.pem
+
+ * The host certificate (`hostcert.pem`) and key (`hostkey.pem`) in `/opt/dirac/etc/grid-security`::
+
+    [dirac@dirac-tuto caUtilities]$ ls /opt/dirac/etc/grid-security/
+    ca  certificates  hostcert.pem  hostkey.pem  openssl_config_host.cnf  request.csr.pem
+
+ * The user credentials for later in `/opt/dirac/user/`::
+
+    [dirac@dirac-tuto caUtilities]$ ls /opt/dirac/user/
+    client.key  client.pem  client.req  openssl_config_user.cnf
+
+--------------------
+Install DIRAC Server
+--------------------
+
+This section is to be run as dirac user.
+
+We will install DIRAC v6r21 with DIRACOS.
+
+First, download the installer, and make it executable::
+
+  mkdir ~/DIRAC && cd ~/DIRAC
+  curl -O -L https://github.com/DIRACGrid/DIRAC/raw/integration/Core/scripts/install_site.sh
+  chmod +x install_site.sh
+
+
+`install_site.sh` requires a configuration file to tell it what and how to install. Create a file called `installation.cfg` with the following content::
+
+  LocalInstallation
+  {
+    #  DIRAC release version to install
+    Release = v6r21p2
```

Comment:
p3

Reply:
Damn, goes fast ! I change it now, but anyway then there will be a revamp needed just before the workshop. 
Btw, if we had a master release every night like we mentioned, we could always use 'master' :-)